appmap/test/test_django.py

```diff
@@ -116,6 +121,21 @@ def test_unnamed(client, events):
     # message attribute.
     assert len(events[0].message) == 0
 
+def test_exception(client, events):
+    try:
+        client.get('/exception')
+
+        assert events[0].http_server_request.items() >= {
+            'request_method': 'GET',
+            'path_info': '/exception',
+            'protocol': 'HTTP/1.1'
+        }.items()
+    except django.core.exceptions.PermissionDenied:
+        assert events[1].parent_id == events[0].id
+        assert events[1].exceptions[0].items() >= {
```

Comment:
To make this check look like others in the file, you should write it as:

```python
assert events[1].items() >= {
   'parent_id': events[0].id,
    'exceptions': [{
        'class': 'django.core.exceptions.PermissionDenied'
    ]}
}.items()
```

To set the `message` attribute of `exceptions[0]`, you need to pass a message to the exception constructor. For example, if you wrote:

```python
def exception_view(_request):
    raise RuntimeError('An error')
```

`exceptions[0].message` should be `'An Error'`.